src/app/services/office.service.ts

```diff
@@ -5,6 +5,7 @@ import { Injectable } from '@angular/core';
 import { Logger } from '@nsalaun/ng-logger';
 import { XMLSerializer } from 'xmldom';
 import * as uniqid from 'uniqid';
+import * as wgxpath from 'wicked-good-xpath';
```

Comment:
Wozu verwenden Sie wgxpath. Es wird nur im Konstruktor einmal initialisiert.

Reply:
wgxpath ist ein Shim, der XPath-Funktionen für IE11 in die bestehenden XML-Klassen einträgt. Mir ging es hauptsächlich um die Funktion `evaluate` in 'OfficeService.findNode', die in IE11 nicht definiert ist.